### PR TITLE
Mod 2 Week 3 Practical Assessment

### PR DESCRIPTION
**Pull request by rtillies**

### CODE REVIEW

File: Mod2/Assessments/Week3Practical.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+1. The count of delayed flights.
+1. The flight numbers, departure city, and arrival city of all flights departing from Denver (`DEN`).
+1. The count of flights departing from Denver (`DEN`).
+1. The sum of prices for all flights arriving to Raleigh-Durham (`RDU`).
+1. The average price for all flights in the database.
+1. The average price for all flights arriving to Raleigh-Durham.
+1. The departure city and number of flights departing from each city.
+1. The count of airlines in the database.
+1. The count of flights in the database.
+1. The flight number, departure city, arrival city, price, and airline name of each flight. Do not return the airline ID number.
```
Comment (zoefarrell) [inline Mod2/Assessments/Week3Practical.md:21]: I definitely like this one, as checking if they can do as basic join and specify what columns to return.

File: Mod2/Assessments/Week3Practical.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+1. Create a new database named `flight_db` using pgAdmin.
+2. Copy [this script](flight_db.sql) and paste it into the query tool to insert records into your database.
+3. Run the following `SELECT` queries individually to get an understanding of the data:
+> `SELECT * FROM airlines;`
+> `SELECT * FROM flights;`
+
+Please provide the QUERY (not the answer) that returns each of the following:
+1. The flight numbers for all delayed flights (i.e. not on time).
+1. The count of delayed flights.
+1. The flight numbers, departure city, and arrival city of all flights departing from Denver (`DEN`).
```
Comment (zoefarrell) [inline Mod2/Assessments/Week3Practical.md:13]: Maybe get rid of this one, feels similar to 1.

File: Mod2/Assessments/Week3Practical.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+1. The count of flights departing from Denver (`DEN`).
```
Comment (zoefarrell) [inline Mod2/Assessments/Week3Practical.md:14]: I'd get rid of this one too for the same reason it feels very similar.

File: Mod2/Assessments/Week3Practical.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+1. The sum of prices for all flights arriving to Raleigh-Durham (`RDU`).
+1. The average price for all flights in the database.
+1. The average price for all flights arriving to Raleigh-Durham.
+1. The departure city and number of flights departing from each city.
+1. The count of airlines in the database.
+1. The count of flights in the database.
+1. The flight number, departure city, arrival city, price, and airline name of each flight. Do not return the airline ID number.
+1. The airline name, flight number, and price of each flight on the Delta airline. (Assume that you do not know the ID number of the Delta airline. In a larger database, you would be expected to memorize ID numbers).
+1. The airline name and number of flights for each airline.
```
Comment (zoefarrell) [inline Mod2/Assessments/Week3Practical.md:23]: Maybe get rid of this one too?

**Comment by zoefarrell** [comment]
This section feels like a great check of the ability to write queries! I marked three that maybe we could remove, 10 feels like a nice round number of questions to go with 10 questions in the other section of the assessment.

I also really like the clear instructions for getting started!

File: Mod2/Assessments/Week3Practical.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Week 3 Assessment
+
+Follow these steps to setup the assessment:
+1. Create a new database named `flight_db` using pgAdmin.
+2. Copy [this script](flight_db.sql) and paste it into the query tool to insert records into your database.
+3. Run the following `SELECT` queries individually to get an understanding of the data:
+> `SELECT * FROM airlines;`
+> `SELECT * FROM flights;`
+
+Please provide the QUERY (not the answer) that returns each of the following:
```
Comment (zoefarrell) [inline Mod2/Assessments/Week3Practical.md:10]: I wonder if we should add explicit instructions to encourage students to try running the queries as they are writing them?

**Comment by memcmahon** [comment]
I think we can leave the questions that seem similar, but do have different commands - we can adjust later if necessary.

**Pull request merged**